samples/callbacks/Version_4/WebSender/App_Start/ServiceBus.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace WebSender
+{
+    using NServiceBus;
+    using NServiceBus.Installation.Environments;
+
+    public class ServiceBus
+    {
+        public static IBus Bus { get; private set; }
+
+        static readonly object padlock = new object();
+
+        public static void Init()
+        {
+            if (Bus != null)
+                return;
+
+            lock (padlock)
+            {
+                if (Bus != null)
+                    return;
+
+                Configure.Serialization.Json();
+                Configure configure = Configure.With();
+                configure.Log4Net();
+                configure.DefineEndpointName("Samples.Callbacks.Sender");
+                configure.DefaultBuilder();
+                configure.InMemorySagaPersister();
+                configure.UseInMemoryTimeoutPersister();
+                configure.InMemorySubscriptionStorage();
+                configure.UseTransport<Msmq>();
+                Bus = configure.UnicastBus().CreateBus().Start(() => configure.ForInstallationOn<Windows>().Install());
```

Comment:
Do you think it does? Bus instance is going to live until the AppDomain shuts down.